### PR TITLE
JAMES-3170 CassandraBlobStoreCache should not propagate failures

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraBlobStoreCache.java [hunk 1]
```diff
@@ -87,7 +87,11 @@
                     .setString(ID, blobId.asString())
                     .setConsistencyLevel(ONE)
                     .setReadTimeoutMillis(readTimeOutFromDataBase))
-            .map(this::toByteArray);
+            .map(this::toByteArray)
+            .onErrorResume(e -> {
+                LOGGER.error("Fail reading blob store cache", e);
```
Comment (rouazana) [inline server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraBlobStoreCache.java:92]: maybe could be a WARN as it is not causing any real issue?

**Comment by chibenwa** [comment]
`MemoryTerminationSubscriberTest.dynamicListeningEventsShouldGetOnlyNewEvents ` unrelated

test this please

**Comment by chibenwa** [comment]
`java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:45377`

Marfff

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Exception in thread "Thread-3101" java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:37861

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**